styles/global.css

```diff
@@ -12,32 +12,6 @@ html {
   src: url(/fonts/homemade-apple-v18.ttf) format('truetype');
 }
 
-* {
-  scrollbar-color: #8885 var(--c-border);
-}
-
-::-webkit-scrollbar {
-  width: 10px;
-}
-
-::-webkit-scrollbar:horizontal {
-  height: 10px;
-}
-
-::-webkit-scrollbar-track {
-  background: var(--c-border);
-  border-radius: 1px;
-}
-
-::-webkit-scrollbar-thumb {
-  background: #8885;
-  border-radius: 1px;
-}
-
-::-webkit-scrollbar-thumb:hover {
-  background: #8886;
-}
```

Comment:
Please revert this. If it's a reason, you maybe bring it in another issue/PR instead.

Reply:
Sorry, I committed this by mistake. I was testing the styles for the native application. I've already reverted it.